core/src/main/java/com/novoda/noplayer/mediaplayer/AndroidMediaPlayerFacade.java

```diff
@@ -117,30 +114,32 @@ private void reportCreationError(Exception ex, Uri videoUri) {
         @Override
         public void onVideoSizeChanged(MediaPlayer mp, int width, int height) {
             if (onSizeChangedListener == null) {
-                return;
+                throw new IllegalStateException("Should bind a OnVideoSizeChangedListener. Cannot forward events.");
```

Comment:
now that we're throwing, it might be good to rename the internal listeners to `forwarders` `OnVideoSizeChangedForwarder` etc